pkgs/development/python-modules/diffusers/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+# propagated build inputs
+, pillow
+, filelock
+, huggingface-hub
+, importlib-metadata
+, numpy
+, regex
+, requests
+, safetensors
+# optional dependencies
+, torch
+, accelerate
+, jinja2
+, datasets
+, protobuf3
+, tensorboard
+}:
+
+buildPythonPackage rec {
+  pname = "diffusers";
+  version = "0.21.4";
+
+  disabled = pythonOlder "3.7.0"; # requires python version >=3.7.0
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3fac38833179427f167c675dda71ee7b4f5e627200457a8d527e40cdaf972688";
```

Comment:
Prefer hash attribute over sha256

Reply:
what is relationship? know i can just rerun it, get error, and change but is it `hash = "sha256-[base64]"`?

Reply:
>  but is it hash = "sha256-[base64]"?

Yes, see https://nixos.org/manual/nixpkgs/unstable/#fetchurl